src/lsm/table.zig

```diff
@@ -21,8 +21,13 @@ pub const TableUsage = enum {
     /// If your usage fits this pattern:
     /// * Only put keys which are not present.
     /// * Only remove keys which are present.
-    /// * TableKey == TableValue (modulo padding, eg CompositeKey)
-    /// Then we can unlock additional optimizations.
+    /// * TableKey == TableValue (modulo padding, eg CompositeKey).
+    /// Then we can unlock additional optimizations:
+    /// * Immediately cancel out tombstone  and the corresponding insert, without waiting for the
```

Comment:
```suggestion
    /// * Immediately cancel out the tombstone and the corresponding insert, without waiting for the
```

Reply:
I hate articles, we do just fine without them in Russian! I think it should be _a_ tombstone and _the_ corresponding insert though? Also sprinkled some more articles for the second bullet point.